docs/source/legacy.rst

```diff
@@ -27,6 +27,24 @@ or
 
 >>> results = simulator.probs_svd(svd, my_detector_list, my_callback)  # my_detector_list can be None
 
+PostSelect API changes
+^^^^^^^^^^^^^^^^^^^^^^
+
+The :code:`PostSelect` class, used to represent a set of post-selection conditions, was moved from Python to C++. This
+change allows to support a richer boolean syntax, with nested condition based on more than the `and` operator.
```

Comment:
This change allows supporting a richer boolean syntax, including nested conditions based on operators in addition to `and`.